platform-enterprise_versioned_docs/version-25.1/upgrade/25.1.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: "25.1"
+description: "Platform Enterprise update guidance"
+date: "21 Aug 2024"
+tags: [enterprise, update, install]
+---
+
+# Upgrade to Seqera Enterprise 25.1
+
+To benefit from the latest fixes and enhancements, we recommend upgrading Seqera regularly.
+
+Seqera supports the latest patch versions of the current and previous major releases (e.g., 25.x and 24.x). Upgrades must proceed one major version at a time.
+
+Alongside standard upgrade steps, some versions may include additional tasks specific to that release.
+
+:::warning[Version compatibility]
+Seqera is backward-compatible with the previous version. As such, existing pipelines, workspaces, and studios will continue to function.
+
+However, default values for components (such as Nextflow) may change between versions. This can require updates to specific pipeline definitions or syntax to ensure compatibility.
+
+Please refer to the [functionality matrix](../functionality_matrix) for detailed information regarding default component versions.
+:::
```

Comment:
Honestly this is more aspirational - I am not close enough on customer upgrades to know if this is fully representative.